src/test/resources/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFs/expected/includeLowQualSites.vcf

```diff
@@ -38,14 +38,14 @@
 ##reference=file:///Users/cnorman/projects/gatk/src/test/resources/large/human_g1k_v37.20.21.fasta
 ##source=GenotypeGVCFs
 #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	GTEX-OXRP-0003	GTEX-QXCU-0004	GTEX-RVPV-0003
-20	10012730	.	G	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012731	.	C	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012732	.	A	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012733	.	G	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012734	.	G	.	0.02	LowQual	DP=4;ExcessHet=3.01;MQ=NaN;RAW_MQ=7200.00	GT:DP:RGQ	0/0:2:6	0/0:0:0	0/0:2:6
-20	10012735	.	C	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012736	.	A	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012737	.	G	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012738	.	C	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012739	.	T	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
-20	10012740	.	C	.	26.18	LowQual	DP=4	GT:AD:DP:RGQ	0/0:2,0:2:6	./.:0,0:0	0/0:2,0:2:6
+20	10012730	.	G	.	.	.	DP=4	GT:DP:RGQ	0/0:2:6	./.:0:0	0/0:2:6
+20	10012731	.	C	.	.	.	DP=4	GT:DP:RGQ	0/0:2:6	./.:0:0	0/0:2:6
+20	10012732	.	A	.	.	.	DP=4	GT:DP:RGQ	0/0:2:6	./.:0:0	0/0:2:6
+20	10012733	.	G	.	.	.	DP=4	GT:DP:RGQ	0/0:2:6	./.:0:0	0/0:2:6
+20	10012734	.	G	.	0.02	LowQual	DP=4;ExcessHet=3.01;MLEAC=.;MLEAF=.;MQ=NaN;RAW_MQ=7200.00	GT:DP:RGQ	0/0:2:6	0/0:0:0	0/0:2:6
```

Comment:
It's weird that this monomorphic site still has a qual, but given that it was already doing thatI don't consider it your responsibility.  I will look into it later.